library/tasks/CreateCountMatrix.wdl

```diff
@@ -86,13 +87,19 @@ task CreateSparseCountMatrix {
   command {
     set -e
 
+
+    #if [[ ${counting_mode} -eq "sn_rna" ]]
+    #then
+    #  counting_mode_arg=" --sn-rna-seq-mode "
+    #fi
+
     CreateCountMatrix \
       --bam-file ${bam_input} \
       --output-prefix sparse_counts \
       --gtf-annotation-file ${gtf_file} \
       --cell-barcode-tag CB \
       --molecule-barcode-tag UB \
-      --gene-id-tag GE
+      --gene-id-tag GE $counting_mode_arg
```

Comment:
We do not need to specify this options because it is not used any more for doing internal, on-the-fly gene name detection for intronic alignments.